components/style/gecko/selector_parser.rs

```diff
@@ -92,6 +94,12 @@ impl ToCss for NonTSPseudoClass {
                         dest.write_str(&value)?;
                         return dest.write_char(')')
                     }, )*
+                    NonTSPseudoClass::Dir(ref dir) => {
+                        dest.write_str(":dir(")?;
+                        // FIXME: This should be escaped as an identifier; see #19231
```

Comment:
This comment should probably go in the `to_css` impl, but that can be done in a followup PR.